lib/cext/ruby.h

```diff
@@ -38,6 +38,16 @@ extern "C" {
 
 #include <truffle.h>
 
+
+// Used in st.h
+#define RUBY_SYMBOL_EXPORT_BEGIN
+#define RUBY_SYMBOL_EXPORT_END
+#define SIZEOF_LONG 8
+#define SIZEOF_LONG_LONG 8
+#define SIZEOF_VOIDP 8
```

Comment:
These are fine as we assume LP64: https://github.com/graalvm/truffleruby/blob/92bcc04488bc52f9eef4fbd6a22ff8ba20e88ec3/doc/contributor/cexts.md#integer-and-pointer-data-model
It might worth a comment though.